rhinosystem/window.py

```diff
@@ -31,20 +30,29 @@ class RhinosystemWindow(Adw.ApplicationWindow):
     __gtype_name__ = 'RhinosystemWindow'
 
     stack_view: Gtk.Stack = Gtk.Template.Child()
-    version: Gtk.Label = Gtk.Template.Child()
+    version: Gtk.Button = Gtk.Template.Child()
+    version_invalid: Gtk.Button = Gtk.Template.Child()
     title: Gtk.Label = Gtk.Template.Child()
 
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self.os_info = SysinfoView()
         self.upgrade_progress = UpgradeView()
+        self.version.connect("clicked", self.copy_version)
         self.os_info.upgrade_button.connect("clicked", self.upgrade_os)
         self.stack_view.add_child(self.os_info)
         self.stack_view.add_child(self.upgrade_progress)
-        self.version.set_label(DeviceInfo().get_os_version())
+        if DeviceInfo().get_os_version() != "Unknown":
+            validation: float = float(DeviceInfo().get_os_version())
+            self.version_invalid.set_visible(False)
+            self.version.set_label(DeviceInfo().get_os_version())
```

Comment:
`DeviceInfo` is a class, not a function, it should be `DeviceInfo.get_os_version()`